cmd/zpool/zpool_main.c

```diff
@@ -1766,7 +1766,7 @@ print_import_config(const char *name, nvlist_t *nv, int namewidth, int depth,
  */
 static void
 print_logs(status_cbdata_t *cb, zpool_handle_t *zhp, nvlist_t *nv,
-    int namewidth, boolean_t verbose, int name_flags)
+    int namewidth, int name_flags)
```

Comment:
The `status_cbdata_t.cb_status` suggestion I do not really follow.  The underlying issue is that `print_logs()` is called in two different paths, from `status_callback()` and then goes to `print_status_config()` or from `show_import()` and then calls `print_import_config()`.  The only reason for `namewidth` and `name_flags` as separate arguments is to be able to handle this.  They also exist in `cb`, so is not really nice.

Another way would be to introduce a structure `import_cbdata_t` for the import case, and then let that and `status_cbdata_t` share a first part structure (holding `namewidth` and `name_flags`), which `print_logs()` can look into.  This could even become handy for `iostat_cbdata_t` too.  Would such type casting "tricks" be allowed in ZFS?  Hmm, if one sends them as two pointers to this function, where one is always `NULL`, one would not even have to resort to any casting.  And depending on if one or the other is non-NULL, one goes to  `print_status_config()` or `print_import_config()`.